src/test/ui/consts/min_const_fn/cmp_fn_pointers.rs

```diff
@@ -1,5 +1,5 @@
 const fn cmp(x: fn(), y: fn()) -> bool { //~ ERROR function pointers in const fn are unstable
-    x == y
+    unsafe { x == y } //~ ERROR pointer operation is unsafe and unsafe operations are not allowed
```

Comment:
Oh nice, where is that coming from? Does `min_const_fn` re-use the same checks as normal unsafety checking?

Reply:
Ah yes, it's actually const checking also looking at the `min_const_fn` flag and not the other way around.